data-es6.js

```diff
@@ -16210,6 +16210,40 @@ exports.tests = [
       },
     },
     {
+      name: 'Number.parseFloat',
+      mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat',
+      exec: function () {/*
+        return typeof Number.parseFloat === 'function';
+      */},
+      res: {
+        babel: true,
+        chrome34: true,
+        edge12: true,
+        es6shim: true,
+        firefox25: true,
+        node0_12: true,
+        safari9: true,
+        typescript: typescript.corejs,
+      },
+    },
+    {
+      name: 'Number.parseInt',
+      mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt',
+      exec: function () {/*
+        return typeof Number.parseInt === 'function';
```

Comment:
this should also test that `Number.parseInt === global.parseInt`, which the spec [requires](https://tc39.github.io/ecma262/#sec-number.parseint)